### PR TITLE
Fixing offline installer redirection values to pick from common.yml from private repo

### PR DESCRIPTION
**Pull request by SMYALTAMASH**

### CODE REVIEW

File: kubernetes/helm_charts/networkconfig/templates/blob-serviceentry.yaml [hunk 1]
```diff
@@ -12,3 +12,11 @@ spec:
     name: blob
     protocol: HTTP
   resolution: DNS
+  hosts:
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-serviceentry.yaml]: Create a separate service entry for this

File: kubernetes/helm_charts/networkconfig/values.j2 [hunk 2]
```diff
@@ -6,6 +6,7 @@ domain_name: {{ domain_name }}
 cloud_storage_url: {{ cloud_storage_url }}
 swarm_kibana_ip: {{groups['swarm-dashboard'][0]}}
 plugin_container_name: {{plugin_container_name}}
-offline_installer_container_name: {{ namespace }}-offlineinstaller
+offline_installer_container_name: {{ offline_installer_container_name }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/values.j2]: Add default as { | d('') }

File: kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml]: @SMY-ALTAMASH Please rename the file to desktop-blob-serviceentry.yaml

File: kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: {{ .Values.namespace }}-offline-blob
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml]: Change this to -desktop-blob

File: kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  namespace: {{ .Values.namespace }}
+spec:
+  hosts:
+  - {{ .Values.offline_cloud_storage_url }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml]: Lets change this to {{ .Values.desktop_blob_url }}

File: kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml [hunk 6]
```diff
@@ -111,19 +111,19 @@ spec:
     - uri:
         prefix: /desktop/
     rewrite:
-      authority: {{ .Values.cloud_storage_url }}
+      authority: {{ .Values.offline_cloud_storage_url }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml]: Change to {{ .Values.desktop_blob_url }}

File: kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml [hunk 7]
```diff
@@ -111,19 +111,19 @@ spec:
...
       uri: /{{ .Values.offline_installer_container_name }}/
     route:
     - destination:
-        host: {{ .Values.cloud_storage_url }}
+        host: {{ .Values.offline_cloud_storage_url }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml]: Change to {{ .Values.desktop_blob_url }}

File: kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml [hunk 8]
```diff
@@ -111,19 +111,19 @@ spec:
     - uri:
         prefix: /desktop/
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml]: Lets move this /desktop/ to a new vs called desktop-blob-vs.yaml

File: kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml [hunk 9]
```diff
@@ -111,19 +111,19 @@ spec:
...
+        host: {{ .Values.offline_cloud_storage_url }}
       headers:
         request:
           add:
             Access-Control-Allow-Methods: GET
             Access-Control-Allow-Origin: '*'
             Authorization: ""
             Cache-Control: public
-            Host: {{ .Values.cloud_storage_url }}
+            Host: {{ .Values.offline_cloud_storage_url }}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml]: Change to {{ .Values.desktop_blob_url }}

File: kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml [hunk 10]
```diff
@@ -111,19 +111,19 @@ spec:
     - uri:
         prefix: /desktop/
     rewrite:
-      authority: {{ .Values.cloud_storage_url }}
+      authority: {{ .Values.offline_cloud_storage_url }}
       uri: /{{ .Values.offline_installer_container_name }}/
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/templates/blob-vs.yaml]: Change this to /{{  .Values.desktop_container_name }}/

File: kubernetes/helm_charts/networkconfig/values.j2 (hunk 2 above)
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/values.j2]: Update on both side to ```desktop_container_name```. If this variable is not specific to kube and is being used in nginx also, then don't change the RHS

File: kubernetes/helm_charts/networkconfig/values.j2 [hunk 11]
```diff
@@ -6,6 +6,7 @@ domain_name: {{ domain_name }}
...
 merge_domain_status: {{ merge_domain_status | lower }}
 merge_proxy_server_name: {{ merge_proxy_server_name | d('""') }}
+offline_cloud_storage_url: {{ sunbird_offline_azure_storage_account | d('"sunbird"') }}.blob.core.windows.net
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/networkconfig/values.j2]: Change the LHS to desktop_blob_url. I belive we are using the RHS in nginx, if so, dont change the RHS

File: kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml (hunk 3 above)
Comment (SMYALTAMASH) [reply kubernetes/helm_charts/networkconfig/templates/blob-serviceentry-offline.yaml]: yup, looks better

File: kubernetes/helm_charts/networkconfig/values.j2 (hunk 11 above)
Comment (SMYALTAMASH) [reply kubernetes/helm_charts/networkconfig/values.j2]: its a new variable all together.

**Pull request merged**